test/integration-metrics.test.js

```diff
@@ -201,6 +207,25 @@ describe('Metrics SDK Integration Tests', function () {
     expect(response.content.mimeType).to.match(/application\/json(;\s?charset=utf-8)?/);
   });
 
+  const authorizationHeader = 'Bearer: a-random-api-key';
+  function getAuthorizationHeader() {
+    if (process.env.SUPPORTS_HASHING) {
+      return 'sha512-7S+L0vUE8Fn6HI3836rtz4b6fVf6H4JFur6SGkOnL3bFpC856+OSZkpIHphZ0ipNO+kUw1ePb5df2iYrNQCpXw==?-key';
+    }
+    return authorizationHeader;
+  }
```

Comment:
Want to move this into the test that's used in?

Reply:
e1d3151a7fbca2d403b0abb1132bfca2603daf70

Reply:
e1d3151a7fbca2d403b0abb1132bfca2603daf70

Reply:
https://github.com/readmeio/metrics-sdks/pull/594/commits/e226af9c6c0729b460272458a72a79c36267b936